libs/db/src/prisma/schema.prisma

```diff
@@ -589,3 +589,11 @@ model AdminActivity {
 
   createdAt DateTime @default(now())
 }
+
+
+/// Not standard SQL practice but whatever, this is a solid way to persist
+/// config
+model Config {
+  id             String @id
+  value          Json
+}
```

Comment:
Missing newline at end of file. My fault, sorry!
Remember to `fixup` this commit, e.g. commit just the fix for this, then
```
git commit -m 'fixup schema'`
git rebase -i HEAD~2
```
then move that commit directly below the schema change commit and `fixup`. Or just use the Fork UI!